docs/user/ppl/functions/math.rst

```diff
@@ -662,3 +662,27 @@ Example::
     | 2.0       | 2.1          |
     +-----------+--------------+
 
+CBRT
```

Comment:
I think the functions in here might be organized in Alphabetical order? It might be good to move this function documentation up.